core/src/test/scala/integration/kafka/api/AbstractConsumerTest.scala

```diff
@@ -66,6 +66,7 @@ abstract class AbstractConsumerTest extends BaseRequestTest {
 
 
   override protected def brokerPropertyOverrides(properties: Properties): Unit = {
+    properties.setProperty(KafkaConfig.NewGroupCoordinatorEnableProp, "true") // enable KIP-848 group coordinator
```

Comment:
At some point we need to test that if the broker's configuration has this set to `false` that the client won't up and explode. That's a future task...